nwb_conversion_tools/datainterfaces/ecephys/blackrock/blackrockdatainterface.py

```diff
@@ -51,18 +51,15 @@ def get_metadata_schema(self):
     def get_metadata(self):
         """Auto-fill as much of the metadata as possible. Must comply with metadata schema."""
         metadata = super().get_metadata()
-
+        metadata["NWBFile"] = dict()
         # Open file and extract headers
         nsx_file = NsxFile(datafile=self.source_data["filename"])
-        session_start_time = nsx_file.basic_header["TimeOrigin"]
-        session_start_time_tzaware = pytz.timezone("EST").localize(session_start_time)
-        comment = nsx_file.basic_header["Comment"]
-
-        # Updates basic metadata from files
-        metadata["NWBFile"] = dict(
-            session_start_time=session_start_time_tzaware.strftime("%Y-%m-%dT%H:%M:%S"),
-            session_description=comment,
-        )
+        if "TimeOrigin" in nsx_file.basic_header:
+            session_start_time = nsx_file.basic_header["TimeOrigin"]
+            session_start_time_tzaware = pytz.timezone("EST").localize(session_start_time)
```

Comment:
why are we assuming EST?

Reply:
as I remember it, there is no tz info in blackrock files, we need to assume something